best-practices/uuid.md

```diff
@@ -0,0 +1,56 @@
+---
+title: UUID Best Practices
+summary: Learn best practice and strategy for using UUID's with TiDB.
+---
+
+# UUID Best Practices
+
+## The usecase for UUID
+
+There are several benefits to using a `UUID` as a primary key, instead of an `AUTO_INCREMENT` integer value:
+
+- They are designed to be globally unique they can be generated on multiple systems without risking conflicts. In some cases, this means that the number of network trips to TiDB can be reduced, leading to improved performance.
+- They are supported by most programming languages and database systems.
+- When used as part of URLs, a UUID is not vulnerable to enumeration attacks. i.e. with an `auto_increment` number, it is possible to guess the other invoice ids or user ids.
```

Comment:
I think it is slightly easier to read as:
```suggestion
There are several benefits to using a `UUID` as a primary key, instead of an `AUTO_INCREMENT` integer value:

- UUIDs are designed to be globally unique, and can be generated on multiple systems without risking conflicts. In some cases, this means that the number of network trips to TiDB can be reduced, leading to improved performance.
- UUIDs are supported by most programming languages and database systems.
- When used as part of URLs, a UUID is not vulnerable to enumeration attacks. i.e. with an `auto_increment` number, it is possible to guess the other invoice ids or user ids.
```

Reply:
Would you also say that an advantage of UUIDs is that in some cases it is less likely to hotspot than an `AUTO_INCREMENT` value? It really depends of course...

Reply:
Not sure about hotspots. That indeed depends on many things. Let's leave it out for now.